tests/Agent/IntegrationTests/IntegrationTests/CodeLevelMetrics/AspNetCoreMvcClmTests.cs

```diff
@@ -13,11 +13,11 @@
 namespace NewRelic.Agent.IntegrationTests.CodeLevelMetrics
 {
     [NetCoreTest]
-    public class AspNetCoreMvcCodeAttributesTest : NewRelicIntegrationTest<RemoteServiceFixtures.AspNetCoreMvcBasicRequestsFixture>
+    public class AspNetCoreMvcClmTests : NewRelicIntegrationTest<RemoteServiceFixtures.AspNetCoreMvcBasicRequestsFixture>
```

Comment:
Not sure I agree with the renaming across the test files/classes, given these are already nested in a `CodeLevelMetrics` folder/namespace (Clm = redundant) and these are testing the `code.*` attributes added for code level metrics... but obviously its up to you and team how you want your code organized.

Reply:
Should be correct now :)